app/pages/ServiceDiscoveryForm/ServiceDiscoveryForm.module.scss

```diff
@@ -111,16 +138,38 @@
         width: 188px;
         border: 1px solid $color-brand;
         border-radius: 3px;
+        &:disabled {
+            background: $color-brand-light;
+            cursor: not-allowed;
+            border: none;
+        }
 
         &:not(:last-child) {
             margin-right: 30px;
+            @media screen and (max-width: $break-tablet-s) {
+                margin-right: 0;
+            }
+        }
+
+        @media screen and (max-width: $break-tablet-s) {
+            width: 250px;
         }
     }
+
     .actionBack {
         background-color: transparent;
         color: $color-brand;
     }
+
     .actionSubmit {
         background-color: $color-brand;
+        @media screen and (max-width: $break-tablet-s) {
+            margin-bottom: 8px;
```

Comment:
This might be a bit too small, since it could lead to users accidentally hitting one button when they meant to hit a different one. From a quick Google search, I found one [blog post](https://uxmovement.com/mobile/optimal-size-and-spacing-for-mobile-buttons/) that recommends between 12–48px spacing, depending on the size of the buttons. I'm not sure if their recommendations for circular buttons translates completely over to our rectangular ones, but I think we could at least bump up the spacing a bit.

Reply:
Thanks for the link. This is great research. I moved the spacing to the parent element `.actionGroup` using `gap` per your great suggestion in a previous PR. I then set the value to 14px. Reading this article makes me think we may even want to bump up the height of these buttons. But I didn't want to change that unilaterally.